styles/main-testing.css

```diff
@@ -0,0 +1,297 @@
+        background-color: darkslategrey;
+        text-align: center;
+    }
+    #bottom-banner *{
+        width: 80%;
+    }
+    #logo-bot{
+        margin-top: 10%;
+    }
+    .footer-text{
+        color: white;
+    }
+    #bot-nav{
+    }
+    #bot-nav li{
+        display: inline;
+        border-left: 2px solid white;
+    }
+    #bot-nav li.first{
+        border-left: none; padding-left: none;
+    }
+    #bot-nav li a{
+        text-decoration: none;
+        color: white;
+        font-size: 1rem;
+    }
+    #sorry{
+        display: none;
+    }
+}
+
+@media only screen and (min-width: 768px){
+    #top-banner{
+        display: grid;
+        grid-template-columns: 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33%;
+        grid-template-rows: 80% 20%;
+        background: url(../images/header-image.jpg) no-repeat center top;
+        background-size: cover;
+        }      
+    /*The grid-template-columns property can also be used to specify the size (width) of the columns.*/
+
+    #logo-top p{
+        font-size: 1rem;
+        color: limegreen;
+        grid-column: 2 / 6;
+        margin-top: -4%;
+    }
+
+    #logo-top{
+        grid-column-start: 2;
+        grid-column-end: 6;
+        text-align: center;
+        align-content: center;
+    }
+    #cart-summary{
+        grid-column-start: 10;
+        grid-column-end: 11;
+        grid-row-start: 1;
+        grid-row-end: 3;
+        display: grid;
+        grid-template-columns: 50% 50%;
+        color: white;
+        align-content: end;
+    }
+    .cart-summary-obj2{
+        width: 33%;
+    }
+    .cart-summary-obj3{
+        grid-row-start: 2;
+        grid-column-start: 1;
+        grid-column-end: 3;
+    }
+    .checkout-btn{
+        width: 100%;
+        color: white;
+        background-color: limegreen;
+        border-color: limegreen;
+        font-size: .75rem;
+        font-weight: bold;
+    }
+    #top-nav{
+        grid-column: 1 / 13;
+        grid-row: 3 / 4;
+        background: white;
+        color: darkslateblue
+    }
+    #top-nav li{
+        display: inline;
+        border-left: 2px solid darkslateblue;
+        padding-left: 0.3em;
+    }
+    #top-nav li.first{
+        border-left: none; 
+        padding-left: none;
+    }
+
+    #main-content{
+        display: grid;
+        grid-template-columns: 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33% 8.33%;
+        background-color: white;
+        justify-content: center;
+    }
+    #about{
+        grid-column: 2 / 12;
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+        align-items: center;
+        padding: 2%;
+
+    }
+    .about-img{
+        grid-column: 1 / 2;
+        padding: 2%;
+    }
+    .about-text{
+        grid-column-start: 2;
+        grid-column-end: 4;
+        padding: 2%;
+        align-items: end;
+    }
+    .about-text-para{
+        width: 100%;
+    }
+
+    #dogs-list{
```

Comment:
You'll need some sort of media query to make these look right on mobile and ipad widths, I think by luck your dogs go into a single column in mobile, but having a media query would help this out.

Reply:
Jimmy,

I'm a little confused by this comment because I have a media query in there for mobile screens. Could you elaborate?

Reply:
Oh, I see it now.

Your media queries are pretty big. It's a good idea to set some baseline styling, and use the media queries for only the few things that will change.